app/kumactl/cmd/inspect/inspect_zones.go

```diff
@@ -75,6 +75,13 @@ func printZoneOverviews(now time.Time, zoneOverviews *system.ZoneOverviewResourc
 				}
 				lastUpdated := util_proto.MustTimestampFromProto(lastSubscription.GetStatus().GetLastUpdateTime())
 
+				var remoteCpVersion string
+				if lastSubscription.GetVersion() != nil {
+					if lastSubscription.Version.KumaCp != nil {
+						remoteCpVersion = lastSubscription.Version.KumaCp.Version
+					}
+				}
+
 				return []string{
 					meta.GetMesh(),                       // MESH
```

Comment:
While you are here changing the code. Mesh should be gone. Zone is Cluster scoped object now.

Reply:
removed